docs/installing.md

```diff
@@ -1,131 +1,93 @@
-# Installation
+# Percona Software Installation
 
+We recommend to install Percona software using the corresponding package manager
+for your system:
 
-It is recommended to install Percona software
-using the corresponding package manager for your system:
-
-
-* **apt** for [Debian and Ubuntu](#install-on-debian-and-ubuntu)
-* **yum** for [Red Hat Enterprise Linux and derivatives
-(including CentOS, Oracle Linux, Amazon Linux AMI, etc.)](#install-on-red-hat-enterprise-linux-and-centos)
+* **apt** for Debian and Ubuntu Linux
+* **yum** for Red Hat Enterprise Linux and compatible derivatives (including CentOS, Oracle Linux, Amazon Linux AMI, etc.)
 
 Find information about supported platforms on the [Percona Release Lifecycle Overview](https://www.percona.com/services/policies/percona-software-support-lifecycle#support).
 
-Alternatively, you can [download percona-release from Percona website](#download-from-percona-website)
-
-=== "Install on Debian and Ubuntu"
+=== "Installation on Debian and Ubuntu Linux"
 
-    If you are running a DEB-based distribution, such as Debian or Ubuntu,
-    use the **apt** package manager to install the `percona-release`
-    official package:
+    If you are running a DEB-based distribution, such as Debian or Ubuntu Linux, use
+    the **apt** package manager to install the `percona-release` official package:
 
-    !!! admonition "Prerequisites"
+    1. Install the `wget` download utility if it's not installed already:
 
-        In Linux distributions that rely on `dpkg`, the packages `wget`, `gnupg2`, `curl` and `lsb-release` are already installed. However, these packages may be missing from Docker base images. In this case, install them manually before running `dpkg`:
-
-        ```
+        ``` sh
         sudo apt update
-        sudo apt install -y wget gnupg2 curl lsb-release 
+        sudo apt install wget 
         ```
 
+    2. Download the `percona-release` repository package:
 
-    1. Fetch the repository package:
-
-        ```
+        ``` sh
         wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb
         ```
 
+    3. Install the downloaded repository package and its dependencies using `apt`:
 
-    2. Install the downloaded repository package with `dpkg`:
-
-        ```
-        sudo dpkg -i percona-release_latest.generic_all.deb
+        ``` sh
+        sudo apt install gnupg2 lsb-release ./percona-release_latest.generic_all.deb
```

Comment:
curl is also needed

Reply:
Are you sure? `percona-release` does not call `curl` anywhere.

Reply:
it calls curl in different places 
https://github.com/percona/percona-repositories/blob/release-1.0-27/scripts/percona-release.sh#L190

Reply:
Ah, I have been checking the version of `percona-release.sh` in the `main` branch. I admit that I find this is very confusing - I would have expected the most recent version of that script to be located in that branch, not a release branch.

Reply:
But if that's the case, shouldn't `curl` be noted as a dependency in the DEB package `control` file?